adcm_aio_client/core/objects/_common.py

```diff
@@ -56,3 +57,9 @@ class WithJobStatus(WithProtectedRequester, AwareOfOwnPath):
     async def get_job_status(self: Self) -> JobStatus:
         response = await self._requester.get(*self.get_own_path())
         return JobStatus(response.as_dict()["status"])
+
+
+class WithImports(WithProtectedRequester, AwareOfOwnPath):
+    @async_cached_property
+    async def imports(self: Self) -> Imports:
+        return Imports(requester=self._requester, path=("imports",))
```

Comment:
Path is incorrect, it should use both `get_own_path` of parent and `"imports"` part